src/lib.rs

```diff
@@ -517,6 +530,25 @@ fn pan_orbit_camera(
         }
 
         // 4 - Update the camera's transform based on current values
+        pan_orbit.radius = pan_orbit.radius.map(|radius| {
+            if pan_orbit.target_radius != radius {
+                // Radius has been changed by zooming
+                has_moved = true;
+                // Smoothing for line-based scrolling
+                let t = 1.0 - pan_orbit.zoom_smoothness;
+                let mut next_radius = radius.lerp(&pan_orbit.target_radius, &t);
+                if approx_equal(next_radius, pan_orbit.target_radius) {
+                    next_radius = pan_orbit.target_radius
+                }
+                if let Projection::Orthographic(ref mut p) = *projection {
```

Comment:
You're only applying the zoom to orthographic cameras for line scrolling, which means pixel scrolling is broken for orthographic cameras.

Reply:
It's not broken. I have added both the values of line scrolling and pixel scrolling to target_radius.